exports_test.go

```diff
@@ -61,7 +61,7 @@ func TestExportsValidation(t *testing.T) {
 	exports.Add(i, i2)
 
 	vr := CreateValidationResults()
-	exports.Validate(vr)
+	_ = exports.Validate(vr)
```

Comment:
remove assignment to nothing.

Reply:
some of my tools started complaining about 'unhandled errors'